Mathlib/Tactic/ENatToNat.lean

```diff
@@ -65,10 +65,11 @@ elab "cases_first_enat" : tactic => focus do
       let name : Name := `enat_to_nat_aux
       setGoals [← g.rename decl.fvarId name]
       let x := mkIdent name
-      evalTactic (← `(tactic| cases' $x:ident using ENat.recTopCoe))
+      evalTactic (← `(tactic| cases $x:ident using ENat.recTopCoe))
```

Comment:
Can you say why this change is fine? To make a guess, is it because it only uses `cases'` with a single ident, and `cases` and `cases'` have the same behaviour in this case? Same question below.

Reply:
Yes, that's correct (and the case names in the second replacement correspond with the given inductor). Where am I supposed to put this information?